src/tagmanager/tm_parser.c

```diff
@@ -951,6 +951,7 @@ static TMParserMapEntry map_POWERSHELL[] = {
 	{'c', tm_tag_class_t},     // class
 	{'i', tm_tag_function_t},  // filter
 	{'g', tm_tag_enum_t},      // enum
+	{'e', tm_tag_undef_t},     // enumlabel
```

Comment:
Should be mapped to `tm_tag_enumerator_t`, shouldn't it?  It's usually mapped to `"Variables"` / `TM_ICON_VAR` apparently (but here enumerators are always gonna have a parent).

Reply:
Done.